### ISSUE TITLE
Issue With Webpack

### ISSUE THREAD

**Issue by stevepop** [phoenixwong/vue2-timepicker#15]
When I compile with webpack I get the following errors;
```
 error  in ./~/vue2-timepicker/src/vue-timepicker.vue

Module build failed: ReferenceError: Unknown plugin "transform-runtime" specified in "/Users/steve/devapps/sites/vue2-naijaprizes/node_modules/vue2-timepicker/.babelrc" at 0, attempted to resolve relative to "/Users/steve/devapps/sites/vue2-naijaprizes/node_modules/vue2-timepicker"
```

**Comment by ksherman** [comment]
I had a similar error! I ended up `npm install`-ing the packages listed in that babelrc file. 

I was going to post an issue on this, any chance these can be added as dependencies in this repo's package.json? I have them as dependencies in my project, I'd rather they be listed with the package if possible.

**Comment by haopei** [comment]
@ksherman, `/vue2-timepicker/.babelrc looks like this`:

```
{
  "presets": ["es2015", "stage-2"],
  "plugins": ["transform-runtime"],
  "comments": false
}

```

Are you referring to `npm install transform-runtime stage-2 es2015`?

**Comment by beijaflor** [comment]
I made quick patch for this problem.

https://github.com/beijaflor/vue2-timepicker

With this, you can import without adding `babel-preset-stage-2` and `babel-plugin-transform-runtime` to your `package.json`
I will refine and make pull request later

**Comment by matrunchyk** [comment]
@beijaflor Thanks for making a correct transpiled build! Could you please make a PR here?
@phoenixwong Could you please accept his PR? :)

**Comment by iamgoodbytes** [comment]
In the meantime, for anyone who's using webpack (like with Laravel 5.4/mix), what would be the way to use this library? Looks like a great component but I'm also stuck with the error @stevepop described. @phoenixwong thanks for the great library and @beijaflor for trying to fix this error!

**Comment by SergioReis97** [comment]
@iamgoodbytes 
```
npm install babel-preset-stage-2 --save
npm install babel-plugin-transform-runtime --save
```
But this adds these dependencies to your project, not this component. I guess it's better to pull the project from  [here](https://github.com/xxRockOnxx/vue2-timepicker) and build it from there since he's trying to mantain the project up somehow.

**Comment by Peace-N** [comment]
Simply run:

```
npm install --save-dev babel-plugin-transform-runtime
npm install --save-dev babel-preset-stage-2
```

**Comment by phoenixwong** [comment]
Sorry for the long wait. Latest release [v0.2.0](https://github.com/phoenixwong/vue2-timepicker/releases/tag/v0.2.0) fixes this issue now (#66)

Please give it a try and feel free to reopen this ticket if the problem still exists.
